packages/gazelle_cli/lib/commands/run/run_project.dart

```diff
@@ -33,14 +33,14 @@ void main(List<String> arguments) async {
     automaticReload: false,
     watchDependencies: false,
     onBeforeReload: (ctx) {
-      // print('Reloading...');
+      if($verbose){
+        print('Reloading...');
+      }
```

Comment:
I would try to replace this `if` statement with an early return.
Right now this is pretty simple, but can become complex in the future.

To keep it readable I would change it like this:
```dart
if (!$verbose) return;
print('Reloading...');
```

Reply:
Yes, you're right. That way the code would look cleaner if we were to add more lines of code to it.
I will edit the pull request today.

Reply:
Actually, instead of using if (verbose) everywhere before printing, it would be nice, if we were to create a function named `verbosePrint` or something, to simplify the code further. 
Now, we can use `verbosePrint` to check for verbose and then print, and use `print` when I want to print irrespective of verbose.

Reply:
`verbosePrint` is a nice touch, good work!